vsdenoise/mvtools.py

```diff
@@ -1056,16 +1060,19 @@ def analyze(
 
         t2 = (self.tr * 2 if self.tr > 1 else self.tr) if self.source_type.is_inter else self.tr
 
+        coherence = lambda blksize: (motion.coherence * blksize * blksize) / 64

```

Comment:
I'm honestly not sure what the desired behavior is here - either we do this calculation or we expect consumers of the library to do the calculation on they're own. But that still leaves questions of how we're supposed to handle refine/recalculate with smaller blocksizes.

I went ahead and added this, as it seems to be the greatest convenience for consumers, but open to feedback here.